### ISSUE TITLE
OSS Examples/ benchmarks/ Feature docs epic

### ISSUE THREAD

**Issue by AyushExel** [lancedb/lancedb#1224]
## Week of 4th Nov:
- [x] Publish building RAG on codebases parts 1&2
- [ ]Write an internal report on finetuning experiments w/ artifacts
- [x] finish writing docs

## Week of 28th oct:
- [x] Wrap up[ docs structure epic](https://github.com/lancedb/lancedb/issues/1476)
- [x] Publish Late chunking blog
- [x] Publish openai swarms agent blog

## Week of 21st oct:
- [x] Video search PoC
- [ ] Discuss case study with Anything LLM

### Week of 14th oct:
- [x] Finetune all llamahub datasets
- [ ] Compare with Nudge-M and Nudge-N
- [x] Publish 2 blogs

### Week of 7th Oct:
- [x] Contextual retrieval blog
- [x] Raptor Blog
- [x] CodeQA end-to-end implementation blog
- [x] Meta<>LanceDB llama quickstart app 2
- [x] SEO next steps
### Week of 30th Sep
- [x] Finish finetuning experiments
- [x] llama notebooks

### Week of 23rd Sep
- [x] Revisit embedding finetuning
- [x] ColPali video search example

### Week of 16th Sep
- [ ] Chunking experiments and report
- [x] Publish ColPali blog/guide & recipe
- [x] Continued work on #1476 (Integrations page)

### Week of 9th Sep
- [x] https://github.com/lancedb/lancedb/issues/1607
- [ ] https://github.com/lancedb/lancedb/issues/1609
- [x] Continued work on https://github.com/lancedb/lancedb/issues/1476 (New RAG guide)
- [x] Backlog from week of 2nd sep

### Week of 2nd Sep
- [x] document parser/recommender with LanceDB (Colpali)
- [ ] Node OSS apps/example
     - [x]  Langhcain ts integration is broken
     - [ ] Llama-index ts integration is broken
- [x] Continued work on https://github.com/lancedb/lancedb/issues/1476
- [x] benchmar reranker latencies
### Week of 26th Aug
- [x] Llama3.1 multi-modal example
- [x] HNSW guide doc
- [ ] Start experiments for report 2 
- [x] Revamp Embedding guide
- [x] https://github.com/lancedb/lancedb/issues/937

### Week of 19th Aug
- [x] Multi-vector reraning docs + examples
- [x] answerai-colbert-small benchmark
- [x] Integrate with answer.ai Rerankers
- [x] https://github.com/lancedb/lancedb/issues/1546
- [x] https://github.com/lancedb/lancedb/issues/1487
- ~[ ] https://github.com/lancedb/lancedb/issues/521~ wontfix
- [ ] https://github.com/lancedb/lancedb/issues/520
### Week of 5th August
- [x] Multi-vector reraning
- [x] Port recipes to docs 
### Week of 29th July
- [x] Revamp example docs
- [x] Finish reranking study
   - [x] Benchmark Squad on Colbert, Crossencoder, Jina, Cohere
   - [x]  add bge reranker
- [x] Improve Phidata integration
### Week of 22nd July
- [x] RRF Reranker
- [x] Reranking benchmark blog.
### Week of 15th July
- [ ] ~CoPali multi-modal embedding function and benchmark~
- [x] scope Q3 ML projects
- [ ] Backlog
### Week of 8th July
- [x] 2 kaggle notebooks
- [x] Wrap up RAG course
- [x] Port Fine-tuning notebook & guide from RAG course to LanceDB docs
- [x] Jina integration docs, ~blog, benchmarks~
- [ ] GraphRAG Docs page

### Week of 1st July
- [x] 2 Kaggle notebooks
- [x] Add fine-tuning section in RAG course
- [x] backlog from previous week

### Week of 24th June
- [ ] Backlog from previous week
- [x] Hybrid search notebook tutorial/benchmark
- [x] Put 2 existing LanceDB examples Kaggle
- [x] Update llama-index, langchain examples in lancedb docs @raghavdixit99 

### Week of 17th June
- [x] 1 SaaS example (wikipedia <> modal) python/ts
- [ ] ~GraphRAG with Kuzu example~ Waiting on updates from prashant
- [x] Multi-Modal model training example to Lance docs
- [x] Multi-Modal dataset creation example to Lance docs
- [x] Model saving, loading and versioning example to Lance docs
- [x] Distributed LLM training

### Week of 10th June
- [x] #1373
- [x] #1374
- [x] Clip training https://github.com/lancedb/lance-deeplearning-recipes/issues/17

# Description

## Experiments
- [x] Diffusion training
- [ ] Saving/Loading model weight chunks using pylance
    - [x] Plugin for vanilla pytorch with features to load, save and version models

## Guides:
- [ ] Fine-tuning embedding models
- [x] Improve retrieval systems 

## Undocumented Features :
- [x] Pydantic optional vector field
- [x] Huggingface integratoin

### Doc improvement tips based on onboarding feedback :

 <h3> Concepts  </h3>

- [x] What is the preferred way to ingest data
      - We support various ways to ingest data - pylist, pydict, arrow, recordbatch, pydantic model list.~ 
       - We don't have an official preferred format.~
- [x] What is the preferred way to define LanceDB table schema**~
        - We allow defining schema explicitly via pyarrow or Pydantic. But we don't say which is preferred ~
       - Pydantic is required for using EMbedding API, which is what we want the users to use ~
**~Better document drop_table~**

- [x] Enrich integrations pages with examples:
   - [x] Langchain
   - [x] Llama-index
 
<h3> Docs typos/bug </h3>
- 
  

### Links
- [x] Langchain  - https://github.com/lancedb/lancedb/issues/1299
- [x] LlamaIndex  - https://github.com/lancedb/lancedb/issues/1300
